mod-orgs/examples/integration_detail_collection.sample

```diff
@@ -0,0 +1,70 @@
+{
+    "integrationDetails": [
+        {
```

Comment:
we have 2 spaces accross 4 in this repository for all other models, so need to follow the same formatting